javascript/code-challenges/challenges-11.test.js

```diff
@@ -0,0 +1,303 @@
+{
+  name: 'bob',
+  age: 32
+}
+
+Becomes:
+[
+<li>name: bob</li>,
+<li>age: 32</li>
+]
+------------------------------------------------------------------------------------------------ */
+
+function transformToLis(obj) {
+  // Solution code here...
+  const liArray = [];
+  for (const [key, value] of Object.entries(obj)) {
+    liArray.push(`<li>${key}: ${value}</li>`);
+  }
+  return liArray;
+}
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 2
+
+Write a function named count that, given an integer and an array of arrays, uses either filter, map, or reduce to count the amount of times the integer is present in the array of arrays.
+
+Note: You might need to use the same method more than once.
+
+For example, count(5, [[1, 3, 5, 7, 9], [5, 5, 5], [1, 2, 3]]) returns 4.
+------------------------------------------------------------------------------------------------ */
+
+const count = (target, input) => {
+  // Solution code here...
+  // let count = 0;
+  // input.map((int1) => {
+  //   int1.map((int2) => {
+  //     if (target === int2) {
+  //       count++;
+  //     }
+  //   });
+  // });
+  // console.log(count);
+  // return count;
+  let acc = 0;
+  input.forEach((arr) => {
+    let count = arr.reduce((a, b) => {
+      return a + (b === target);
+    }, 0);
+    acc += count;
+  });
+  return acc;
+};
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 3
+
+Write a function that, given an array of integer arrays as input, calculates the total sum of all the elements in the array.
+
+You may want to use filter, map, or reduce for this problem, but are not required to. You may need to use the same method more than once.
+
+For example, [[1, 2, 3, 4, 5], [6, 7, 2, 4, 5, 7], [9, 2, 3, 6,]] returns 66.
+------------------------------------------------------------------------------------------------ */
+
+const totalSum = (input) => {
+  // Solution code here...
+  let total = 0;
+  input.forEach((arr) => {
+    let sum = arr.reduce((a, b) => {
+      return (a += b);
+    }, 0);
+    total += sum;
+  });
+  return total;
+};
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 4
+
+Write a function named divisibleByFiveTwoToThePower that accepts an array of arrays as input.
+
+This function should first remove any elements that are not numbers or are not divisible by five.
+
+This function should then raise 2 to the power of the resulting numbers, returning an array of arrays.
+
+For example, [ [0,2,5,4], [2,4,10], [] ] should return [ [1, 32], [1024], [] ].
+------------------------------------------------------------------------------------------------ */
+
+const divisibleByFiveTwoToThePower = (input) => {
+  // Solution code here...
+  // let filteredArray = input.map((array) =>
+  //   array.filter((number) => number % 5 === 0)
+  // );
```

Comment:
Same here, removal unused code to keep the rest of this nice and readable/clean 👍